src/main/java/step1/StringCalculator.java

```diff
@@ -0,0 +1,111 @@
+package step1;
+
+import java.util.ArrayList;
+
+import static step1.Utils.ValidationUtils.isBlankThrowException;
+import static step1.Utils.ValidationUtils.isNotNumericThrowException;
+
+public class StringCalculator {
+
+    private static final String PLUS = "+";
+    private static final String MINUS = "-";
+    private static final String MULTIPLY = "*";
+    private static final String DIVIDE = "/";
+    private static final String SPACE_DELIMITER = " ";
+
+    private ArrayList<String> operands;
+    private ArrayList<Integer> operates;
+    private int result;
+
+    StringCalculator(){
+        result = 0;
+        operands = new ArrayList<>();
+        operates = new ArrayList<>();
+    }
+
+    public int calculate(String input) {
+        isBlankThrowException(input);
+        String[] splitedStringArray = splitStringBySpace(input);
+
+        iterateClassification(splitedStringArray);
+        iterateCalculation();
+        return result;
+    }
+
+    public String[] splitStringBySpace(String input){
```

Comment:
```suggestion
    private String[] splitStringBySpace(String input){
```

내부에서만 사용하는 경우 접근제어자를 private으로 구성해주세요